src/strchr.rs

```diff
@@ -20,42 +20,93 @@ pub unsafe extern "C" fn strchr(haystack: *const CChar, needle: CInt) -> *const
 	core::ptr::null()
 }
 
+/// Rust implementation of C library function `strrchr`
+#[no_mangle]
+pub unsafe extern "C" fn strrchr(haystack: *const CChar, needle: CInt) -> *const CChar {
```

Comment:
I'd prefer to stick to one-function-per-module as it makes it easier to gate which global symbols we export. Every function should have its own module and its own feature.